app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -850,11 +908,10 @@ class NewMessageViewModel @Inject constructor(
         if (!isFinishing) {
             copyFromRealm()
                 .apply {
```

Comment:
Why do we do this `.apply { … }` ? 🤔

The `uiBody` assignation was used for the `saveSnapshot()` call, but the `uiSignature` & `uiQuote` ? 🤔